sechub-analyzer-cli/src/main/java/com/daimler/sechub/analyzer/cli/Analyzer.java

```diff
@@ -0,0 +1,137 @@
+package com.daimler.sechub.analyzer.cli;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.List;
+
+import org.apache.commons.cli.CommandLine;
```

Comment:
I actually did not use Log4j 1.x, but used instead Log4j 2. Log4j 2 separates the implementation from the API, therefore it would be possible to use the Log4j 2 API with a different implementation. 

Log4j 2 incorporated the good parts from SLF4J. Furthermore, it made it possible to log objects instead of only strings:

https://stackoverflow.com/questions/41498021/is-it-worth-to-use-slf4j-with-log4j2